components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/services/ConditionalAuthenticationMgtService.java

```diff
@@ -44,7 +44,8 @@ public class ConditionalAuthenticationMgtService extends AbstractAdmin {
      */
     public String[] getAllAvailableFunctions() {
 
-        List<String> jsFunctions = Stream.of("executeStep", "selectAcrFrom", "sendError", "Log.info", "require").collect
+        List<String> jsFunctions =
+                Stream.of("executeStep", "selectAcrFrom", "sendError", "Log.info", "require", "secrets").collect
```

Comment:
Can we add a comment on this method how this differs from rest of the functions or what exactly is achieved by adding functions here

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5723/commits/0b71da48f757b7651d5749723d7ed7b024393b6f